edgelet/contrib/config/linux/default.toml

```diff
@@ -0,0 +1,26 @@
+# hostname = "my-device"
+
+# parent_hostname = "my-parent-device"
+
+# trust_bundle_cert = "file:///path/to/file.pem"
+
+homedir = "/var/lib/aziot/edged"
+
+[agent]
+name = "edgeAgent"
+type = "docker"
+
+[agent.config]
+image = "mcr.microsoft.com/azureiotedge-agent:1.0"
+
+[connect]
+workload_uri = "@connect_workload_uri@"
+management_uri = "@connect_management_uri@"
+
+[listen]
+workload_uri = "@listen_workload_uri@"
+management_uri = "@listen_workload_uri@"
```

Comment:
This should be @listen_management_uri@

(The 404 error for /trust-bundle I was getting was caused by this)

Reply:
Ah shit. I fixed that in template but forgot to fix it here. Fixed now.